lapack/testlapack/dlahr2.go

```diff
@@ -203,7 +203,10 @@ func Dlahr2Test(t *testing.T, impl Dlahr2er) {
 	defer r.Close()
 
 	var tests []Dlahr2test
-	json.NewDecoder(r).Decode(&tests)
+	err = json.NewDecoder(r).Decode(&tests)
```

Comment:
Eventually I'd like to get rid of these json-based tests.

Reply:
Yes. That would be good.